shlr/ar/ar.c

```diff
@@ -212,16 +208,82 @@ static int ar_parse_header(RzArFp *arf, filetable *tbl, ut64 arsize) {
 }
 #undef VERIFY_AR_NUM_FIELD
 
+/**
+ * \brief Open specific file withen a ar/lib file.
+ * \param arname the name of the .a file
+ * \return a list of files or NULL
+ *
+ * Open an ar/lib and returns all the object files inside it.
+ */
+RZ_API RzList *ar_open_all(const char *arname, int perm) {
```

Comment:
Can be the `arname` NULL?

Reply:
it should not, but it will fail when opening the file.